dora/core/client/fs/src/main/java/alluxio/client/file/cache/CacheManager.java

```diff
@@ -333,4 +334,17 @@ default void invalidate(Predicate<PageInfo> predicate) {
 
   @Override
   Optional<CacheUsage> getUsage();
+
+  /**
+   * Get a {@link DataFileChannel} which wraps a {@link io.netty.channel.FileRegion}.
+   * @param pageId the page id
+   * @param pageOffset the offset inside the page
+   * @param bytesToRead the bytes to read
+   * @param cacheContext the cache context
+   * @return an object of {@link DataFileChannel}
+   */
+  default DataFileChannel getDataFileChannel(
+      PageId pageId, int pageOffset, int bytesToRead, CacheContext cacheContext) {
+    throw new UnsupportedOperationException();
```

Comment:
do you think it should be a method that must be implemented? since important and should be impl?

Reply:
Updated. But I am not sure if there would be other type of CacheManager that can't create a DataFileChannel in the future.